### PR TITLE
Chapter 5

### PR DESCRIPTION
**Pull request by ckvvanle**

### CODE REVIEW

**Comment by ckvvanle** [comment]
chapter_5

**Comment by ckvvanle** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (hhhhuy112) [inline app/assets/stylesheets/users.scss]: remove this file if you don't use it

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <%= link_to "Ruby on Rails Tutorial", "http://www.railstutorial.org/" %>
```
Comment (hhhhuy112) [inline app/views/layouts/_footer.html.erb]: Please use I18n with pass variable

**Comment by hhhhuy112** [comment]
commented

**Comment by ckvvanle** [comment]
chapter_5

**Comment by ckvvanle** [comment]
ready

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
-  <%= t ".p1" %>
-  <%= link_to raw(t ".body_link"), t(".link") %>
-  <%= t ".p3" %>
-</p>
+<div class="center jumbotron">
+  <h1><%= t ".h1" %></h1>
+  <h2>
+    <%= t ".p1" %>
+    <%= link_to raw(t ".body_link_book"), t(".link_book") %>
+    <%= t ".p3" %>
```
Comment (chien-pv) [inline app/views/static_pages/home.html.erb]: Đặt name i18n lại cho dễ hiểu nhé p1, p3 thế sao hiểu được

File: app/assets/javascripts/users.coffee [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (chien-pv) [inline app/assets/javascripts/users.coffee]: Xóa file này đi nếu không dùng

**Comment by chien-pv** [comment]
commented

**Comment by ckvvanle** [comment]
chapter_5

**Comment by ckvvanle** [comment]
ready

File: app/views/static_pages/about.html.erb [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-  <%= link_to raw(t ".body_link_2"), t(".link_2") %>
-  <%= t ".p3" %>
-  <%= link_to raw(t ".body_link_3"), t(".link_3") %>
-  <%= t ".p4" %>
-  <%= link_to raw(t ".body_link_4"), t(".link_4") %>
-  <%= t ".p5" %>
+  <%= t ".text_1" %>
+  <%= link_to raw(t ".body_link_1"), "http://www.railstutorial.org/" %>
+  <%= t ".text_2" %>
+  <%= link_to raw(t ".body_link_2"), "http://www.railstutorial.org/book" %>
```
Comment (hhhhuy112) [inline app/views/static_pages/about.html.erb]: don't need use raw   <%= link_to (t ".body_link_2"), "http://www.railstutorial.org/book" %>

**Comment by hhhhuy112** [comment]
commented

**Comment by ckvvanle** [comment]
chapter_5

**Comment by ckvvanle** [comment]
ready

**Comment by hhhhuy112** [comment]
merged
You can merge it

**Pull request merged**